client/src/components/campaignCreatorComponents/CampaignCreator.tsx

```diff
@@ -36,8 +43,81 @@ const CampaignCreator = ({ campaignSettings, setCampaignSettings }: CampaignCrea
         navigate('/twilight-library/dashboard');
     };
 
+    const addToTimeline = (
+        timeline: Record<number, string[]>,
+        nodeKey: keyof NodePillarLists,
+        typeKey: keyof TimelineOptionList,
+    ) => {
+        if (!selectedCampaign.flexible_nemesis_encounters && typeKey === 'nemesis') {
+            // Handle strict Nemesis timeline
+            const nemesis_tier: string = nodeKey.slice(-1);
+            selectedCampaign[nodeKey].forEach((selection: string) => {
+                [1, 2, 3].forEach((level: number) => {
```

Comment:
`Array.from({ length : 3 }).forEach`

Reply:
in any other situation I'd agree. For this instance, the 1,2,3 correlate to monster level and is more readable for me here.